Makefile

```diff
@@ -235,7 +235,7 @@ run_full_flow: run deploy_nodes set_dns
 
```

Comment:
Please remove from `run_full_flow` as well

Reply:
`run_full_flow` invokes `deploy_nodes`, which doesn't have `set_dns`. 

Actually, I'm not sure how it's supposed to work because deploying nodes without installation won't have an API VIP yet. Maybe that's exactly the catch in #836 - there won't be API VIP connectivity after deploying nodes using `make` and then installing them manually from the GUI or API. On the other hand, it didn't work correctly anyway. 

Maybe we can open an enhancement to re-introduce updating dns-host in libvirt from `set_dns`, but this time with a more complex logic that gets API VIP from the cluster.

Reply:
For now I added an explanation to the commit message. This issue can be considered a limitation and won't affect automated flows.